### PR TITLE
[feature] Support importing MySQL geographic data types and PostgreSQL array data types

### PR DESCRIPTION
**Pull request by vinlee19**
1. The Geometry data type in MySQL will be converted to a JSON-formatted string. For example, the data `ST_GeomFromText('POINT(3 1)') ` will be converted to a JSON string:
```
{"coordinates":[3,1],"type":"Point","srid":0}
```

2. Support importing array data types in PostgreSQL.

### CODE REVIEW

**Review by DongLiang-0 — COMMENTED** [review]
Can we add a unit test case?

**Comment by vinlee19** [comment]
> Can we add a unit test case?
done

File: src/main/java/org/apache/doris/kafka/connector/converter/type/doris/DorisType.java
Comment (DongLiang-0) [inline src/main/java/org/apache/doris/kafka/connector/converter/type/doris/DorisType.java:1]: Are the changes in this file a bit redundant? I didn’t find where to quote them.

File: src/main/java/org/apache/doris/kafka/connector/converter/type/debezium/ArrayType.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        if (sourceValue == null) {
+            return null;
+        }
+        Schema valueSchema = schema.valueSchema();
+        String type =
+                Objects.nonNull(valueSchema.name())
+                        ? valueSchema.name()
+                        : valueSchema.type().name();
+
+        if (sourceValue instanceof ArrayList) {
```
Comment (DongLiang-0) [inline src/main/java/org/apache/doris/kafka/connector/converter/type/debezium/ArrayType.java:77]: Would it be better to change this to ```sourceValue instanceof List```?

**Review by DongLiang-0 — APPROVED** [review]
LGTM

**Review by DongLiang-0 — APPROVED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**